### DISCUSSION TITLE
how to use shadow on text?

### DISCUSSION THREAD

**Discussion by soerjo** [tailwindlabs/tailwindcss#9648] [Help]
I can't use drop-shadow or shadow to give shadow in my text

**Comment by wongjn** [comment]
You could use an [arbitrary property](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties) for [text-shadow](https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow), like `[text-shadow:red_1px_2px_4px]`.

**Reply by soerjo** [reply]
wow, thanks. it works.
